README.md

```diff
@@ -24,7 +24,7 @@ These instructions will get you a copy of the role for your Ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.4.3.0 version installed.
+Ansible 2.7.9.0 version installed.
```

Comment:
I think that the possibility to install MariaDB with this role should be defined at the beginning in this README file

Reply:
Done!